libraries/permissions/impl/src/main/kotlin/io/element/android/libraries/permissions/impl/PermissionsStore.kt

```diff
@@ -16,17 +16,3 @@
 
 package io.element.android.libraries.permissions.impl
 
-import kotlinx.coroutines.flow.Flow
```

Comment:
This file can be removed. AS "move" action sometimes does not move the file but just the content, not sure why. It can happen when there are other items in the file, but it's not the case here...